packages/edit-site/src/components/sidebar-navigation-screen-template/index.js

```diff
@@ -29,6 +29,7 @@ export default function SidebarNavigationScreenTemplate() {
 			actions={
 				<Button
 					variant="primary"
+					className="edit-site-sidebar-navigation-screen__button"
```

Comment:
```suggestion
					className="edit-site-sidebar-navigation-screen__edit"
```

Let's rename to match a bit more the other classes we have in sidebar(ex: `edit-site-sidebar-navigation-screen__back`).

Reply:
Thanks @ntsekouras 🙏 

I have updated the PR with rename of the class and added the same class here: https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-site/src/components/sidebar-navigation-screen-navigation-item/index.js#L46